zstacklib/zstacklib/gpu/base.py

```
@@ -0,0 +1,512 @@
+        
+        Args:
+            pci_device_to: The PCI device transfer object
+        """
+        pass
+    
+    # ==========================================================================
+    # Device Type Validation
+    # ==========================================================================
+    
+    @classmethod
+    def is_valid_device(cls, device_name, device_type):
```

Ruff (0.14.13)

Unused class method argument: `device_name`

(ARG003)